### ISSUE TITLE
automatic_installation mason-null-ls not working

### ISSUE THREAD

**Issue by jay-babu** [AstroNvim/AstroNvim#1533]
### Checklist

- [X] Using a stable version of Neovim (i.e. not neovim nightly)
- [X] `:PackerSync`
- [X] `:AstroUpdate`
- [X] Restarted AstroNvim

### AstroNvim version

2.11.8

### Neovim version (>= 0.8)

0.8

### Operating system/version

macOS

### Describe the bug

Automatic installation isn't working. Passing in arguments with via ensure installed works fine though.

### Steps to Reproduce

Add some tools in Null ls. Enable automatic installation in mason-null-ls and it doesn't work.

### Expected behavior

Tools are automatically installed

**Comment by mehalter** [comment]
Thanks for reporting, I have fixed this in v3 which will release in a couple weeks :)

**Comment by jay-babu** [comment]
Thanks!

I started slowly moving my config to v3. Seems like it will take awhile, but def worth it

**Comment by mehalter** [comment]
Since this has been  resolved in the new AstroNvim v3.0 and it's available on `nightly` I'm going to go ahead and close this as completed! Thanks again for reporting 😄

**Comment by mangkoran** [comment]
Hello. It seems I also encountered this issue on freshly installed AstroVim 3.22.0. Below is my `plugins/user.lua`.

```
return {
  {
    "williamboman/mason-lspconfig.nvim",
    opts = {
      automatic_installation = true,
    }
  },
  {
    "neovim/nvim-lspconfig",
    opts = {
      servers = {
        gopls = {},
        clangd = {},
        lua_ls = {},
      }
    }
  },
}
```
